### ISSUE TITLE
Domain options 3 dots icon is difficult for accessibility and easy to misclick

### ISSUE THREAD

**Issue by kuropixel** [Automattic/wp-calypso#56230]
### What

There needs to be a bigger area or button instead of the smaller "3 dots" icon on the Domains page for options.

### Why

I have found that the method for changing a primary address involves going through the tiny 3 dots icon on the Domains page. Even when taking a screenshot to guide others, I have sometimes misclicked and end up getting taken to the detailed domain page by accident. For accessibility, having such a small area that you need to click to access the domain settings might be difficult for some users, especially those with challenges for movement or those with limited tech experience. 

![domains page setting](https://user-images.githubusercontent.com/83020905/133186807-b2d04471-dbbb-4b53-bf06-059fcfc0608e.png)

I think having a clickable button instead would be an accessibility improvement because it would give users a larger clickable area. Another option could be to not take the user to the deeper domain details page if they misclick near the 3 dots icon.

### How

![example button](https://user-images.githubusercontent.com/83020905/133187110-e4bf983b-26c6-4bc4-a2ae-4831ace9c678.png)

**Comment by kelliepeterson** [comment]
@poligilad-auto Thoughts?

**Comment by poligilad-auto** [comment]
Thanks for bringing this to our attention @kuropixel.

@kelliepeterson in the new table design, not all the domain area will be clickable. I think clicking on the domain name or on an action in the "3 dots" should take you to the internal domain page, so we can eliminate clicking on the domain by mistake in that way. This will also make the "3 dots" more accessible as you will see that it's clickable when hovering about it (vs. now that the whole block is clickable).

The "3 dots" menu is a common UI pattern in many products, including in various pages of WordPress .com and even here in Github.

**Comment by kuropixel** [comment]
@poligilad-auto Thank you for your reply. I'm not against the "3 dots" menu. Hearing the table design may help with this issue is great. I had certain types of customers as well as my own parents in mind when I was checking the domains page. Appreciate the info!

**Comment by poligilad-auto** [comment]
@kuropixel of course and please keep this feedback coming! It always helps us improve :)

**Comment by obenland** [comment]
It looks like focus styles could use a pass as well, there are some inconsistencies.

Marked as Good First Issue since these should mostly be style changes and could be a good way to learn the process of creating a Calypso PR.

**Comment by jordesign** [comment]
It is worth noting - that this domain panel now has less horizontal space (as it has been redesigned to include more items).

There is still scope for some improvement though.
<img width="1009" alt="Screen Shot 2023-01-04 at 11 15 43 am" src="https://user-images.githubusercontent.com/1842363/210463617-028ac7fb-7758-4bc6-a95f-841ae3435363.png">

**Comment by jordesign** [comment]
@kuropixel I've got a Pull Request that addresses this:
https://github.com/Automattic/wp-calypso/pull/71656

you can test it out at this URL (if it fails then you can get a new link from the PR itself).
https://container-upbeat-lumiere.calypso.live/

**Comment by kuropixel** [comment]
@jordesign Thanks for sharing! Seems great to me and surely goes a long way in being more accessible for those that need it.

**Comment by jordesign** [comment]
Closing this as https://github.com/Automattic/wp-calypso/pull/71656 is now deployed
